Automattic/woocommerce-payments
# Add a request signature to user/exists platform checkout endpoint.

#### Changes proposed in this Pull Request

This change adds a few necessary pieces of information when checking if a platform checkout customer exists.

#### Testing instructions

* Apply this patch.
* Fill out the email address field on the shortcode checkout.
* You should see the two new items in the request payload.

-------------------


**Post merge**
